index.js

```diff
@@ -127,7 +127,9 @@ module.exports = {
             checkString(value, node);
           },
           JSONLiteral(node){
-            const { value } = node;
+            const { value, parent } = node;
+            const nodeIsIdentifier = parent.type === 'JSONProperty' && parent.key === node
+            if (nodeIsIdentifier) return;
```

Comment:
If you want to exclude the JavaScript property name from the check as well, you'll need to make the same changes to the JavaScript check method.

Reply:
Thank you for the help @ota-meshi! @constgen if you'd like to include that check in the PR, let me know.

Reply:
Now I realized. These two JS expressions produce different AST nodes:

1. `{ name: 'value' }` - `name` is Identifier type
2. `{ 'name': 'value' }` - `'name'` is Literal type

So I am going to add additional checks for JS too

Reply:
Excluded property names from parsing and added a test for this. But tests are not run automatically on any CI/CD